portfolio/src/main/webapp/script.js

```diff
@@ -40,15 +37,12 @@ function changeBackgroundColor(color){
     document.body.style.background = color;
 }
 
-function getGreeting(){
-    fetch('/data').then(response => response.json()).then((greetings) => {
-        console.log(greetings);
-        const greetingContainer = document.getElementById('greeting-container');
-        greetings.forEach((greeting) => {
-            console.log(greeting);
-            greetingContainer.appendChild(createListElement(greeting));
+function getComments(){
+    fetch('/comments').then(response => response.json()).then((comments) => {
```

Comment:
Might try some newlines to clean up the formatting here (for instance get the `.then`s stacked vertically). It's a bit hard to pick out what's going on at first glance.